ProjectMarsLanguageFeature/StepDefinitions/LanguageAddStepDefinitions.cs

```diff
@@ -63,15 +64,15 @@ public void ThenShouldNotAdd(string language)
             {
                 Thread.Sleep(5000);
                 IWebElement languageRead = driver.FindElement(By.XPath("//div/section[2]/div/div/div/div[3]/form/div[2]/div/div[2]/div/table/tbody[last()]/tr/td[1]"));
-                Assert.That(!string.IsNullOrEmpty(languageRead.Text), "Language should not be added");
+                Assert.That(!string.IsNullOrEmpty(languageRead.Text), "Space should not be added");
```

Comment:
please move all webelements to pages, thie find by element logic lives in pages not in steps! fix this everywhere, also you can create a method in pages that would return the result to be used for assertion, make sure you do not return a IWebElement but the assert value itself

Reply:
Sure